bot/constants.py

```diff
@@ -84,6 +85,7 @@ class Client(NamedTuple):
     token = environ.get("SEASONALBOT_TOKEN")
     sentry_dsn = environ.get("SEASONALBOT_SENTRY_DSN")
     debug = environ.get("SEASONALBOT_DEBUG", "").lower() == "true"
+    github_bot_repo = "https://github.com/python-discord/bot"
```

Comment:
This isn't the correct repository.
```suggestion
    github_bot_repo = "https://github.com/python-discord/seasonalbot"
```